packages/gatsby/cache-dir/navigation.js

```diff
@@ -159,6 +158,55 @@ function init() {
   maybeRedirect(window.location.pathname)
 }
 
+class RouteAnnouncer extends React.Component {
+  constructor(props) {
+    super(props)
+    this.announcementRef = React.createRef()
+  }
```

Comment:
ES6 tip (feel free to do this if you want). But you can ditch the constructor here so we're just instantiating a class variable. Do it like this instead:

```js
class Router {
  announcementRef = React.createRef()
}
```

It'll achieve the same result! (if this doesn't work ignore, it does require a specific babel setup to make it work and maybe we don't have that)

Reply:
I think I'm going to leave this for a later iteration. I'm ready to get this thing OUT!